community/scroll/sources

```diff
@@ -0,0 +1 @@
+git+git://git.suckless.org/scroll
```

Comment:
Any reason for using a git source?

Reply:
As far as i could tell, suckless.org does not maintain a release tarball on their site for this program. although, im still a newb and may have missed something obvious :). 

src: https://dl.suckless.org

Reply:
Alrighty.